storage/storage.go

```diff
@@ -143,17 +143,32 @@ func getDataSourceFromConfig(configuration Configuration) (string, error) {
 
 // Init method is doing initialization like creating tables in underlying database
 func (storage DBStorage) Init() error {
-	_, err := storage.connection.Exec(`
-		create table IF NOT EXISTS report (
-			org_id      integer not null,
-			cluster     varchar not null unique,
-			report      varchar not null,
-			reported_at datetime,
-			last_checked_at datetime,
-			PRIMARY KEY(org_id, cluster)
-		);
-	`)
-	return err
+	if err := InitMigrationInfo(&storage); err != nil {
+		return err
+	}
+
+	ClearMigrations()
+
+	AddMigration(Migration{
```

Comment:
I'm not sure about defining migration in `Init` maybe we could move it somewhere else?

Reply:
@bond95 I wanted to put it into `init()` in `migration.go`, but that breaks tests that modify the migration list. Perhaps it shouldn't even be possible to add migrations at runtime ( = they'd be hardcoded in `migration.go` or their own file in the `storage` package and there would be no way to modify the migration list externally at runtime).

Reply:
Yes, I think migration should be hardcoded into binary, and may be changed only from point in code.

Reply:
@bond95 So do you want me to remove the `AddMigration` and `ClearMigrations`, including their respective tests?

Reply:
@bond95 So do you want me to remove the `AddMigration` and `ClearMigrations`, including their respective tests?

Reply:
@natiiix no, I just want to move it in separate file.